projects/schema-form/src/lib/form.component.ts

```diff
@@ -57,7 +58,7 @@ export function useFactory(schemaValidatorFactory, validatorRegistry, propertyBi
 })
 export class FormComponent implements OnChanges, ControlValueAccessor {
 
-  @Input() schema: any = null;
+  @Input() schema: ISchema = null;
```

Comment:
It will probably break when using `stringNullCheck` in tsconfig, as `null` is not a valid value for `ISchema`, so better to write:
```typescript
@Input() schema: ISchema | null = null;
```

Reply:
changed it, thanks!